src/cascadia/UIHelpers/UIHelpers.vcxproj

```diff
@@ -64,15 +71,15 @@
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
     </ProjectReference>
   </ItemGroup>
-
+  <ItemGroup>
+    <PRIResource Include="Resources\en-US\Resources.resw" />
+  </ItemGroup>

```

Comment:
Double-checking: Just `en-US`?

Reply:
Yes, but the `<OCResourceDirectory>` key was missing.